sherlock/resources/data.json

```diff
@@ -66,8 +66,8 @@
   "9GAG": {
     "errorType": "status_code",
     "rank": 389,
-    "url": "https://www.9gag.com/u/{}",
```

Comment:
Doesnt removing that `www` break everything? I think you might have accidently misunderstood my last comment